scripts/components/PaymentForm.jsx

```diff
@@ -0,0 +1,112 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var DatePicker = mui.DatePicker;
+var TextField = mui.TextField;
+var RaisedButton = mui.RaisedButton;
+var Paper = mui.Paper;
+
+var PaymentForm = React.createClass({
+  getInitialState() {
+    return {
+      date: '', // user entered date
+      property: '', // user entered property address
+      amount: '', // user entered amount
+    }
+  },
+
+  propTypes: {
+        newDataAdded:   React.PropTypes.func,
+  },
+
+  // Capture the input field state after each keypress.
+  onChange(field, event) {
+    this.setState({ [field]: event.target.value });
+  },
+
+  // Handle the form submission event when the user adds new payment.
+  onSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+      // API call to add payment.
+    Api.addPayment({
+      data: {
+        date: this.refs.DatePicker.getDate(),
+        property: this.state.property,
+        amount: this.state.amount
+      },
+      callback: (err, response) => {
+        if (err) {
+          return;
+        }
+        // Clear the form
+        this.setState({
+          date: "",
```

Comment:
To maintain consistent styling with the codebase, when writing strings in JavaScript we want to use single quotes, not double quotes.
```javascript
date: '',
property: '',
```
Double quotes are fine for props/attributes in JSX / HTML